src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -101,6 +113,12 @@ class TooltipRenderedOnPageBody extends React.PureComponent {
         });
     }
 
+    measureContent({nativeEvent}) {
+        this.setState({
+            tooltipContentWidth: nativeEvent.layout.width,
+        });
+    }
+
```

Comment:
A new function is not required here, remove it and use `updateTooltipTextWidth` instead after renaming it to `updateTooltipContentWidth`.